README.md

```diff
@@ -166,6 +166,7 @@ summary (optional)
 images (optional, if none provided defaults to socialBanner in siteMetadata config)
 authors (optional list which should correspond to the file names in `data/authors`. Uses `default` if none is specified)
 layout (optional list which should correspond to the file names in `data/layouts`)
+canonical (optional, canonical url for the post for SEO)
```

Comment:
Your code uses `canonicalUrl`, not `canonical`. Can you update the rest of the documentation to use `canonicalUrl`? Thanks.